test/schema.spec.js

```diff
@@ -272,6 +275,16 @@ describe('Schema Tests', function() {
 				});
 			});
 
+			it('set the defaults for properties of type array', function(done) {
+				this.schema.create({
+					ofObjects: [{fieldA: '1234'}] 
+				}).then(function(instance) {
+					instance.should.have.deep.property('ofObjects[0].fieldA', '1234');
+					instance.should.have.deep.property('ofObjects[0].fieldB', 100);
+					done();
+				}).catch(done);
```

Comment:
We are we using the **done** keyword inside the catch? If schema.create() throws an exception will the test pass?